Tools/AP_Periph/AP_Periph.cpp

```diff
@@ -181,6 +181,7 @@ void AP_Periph_FW::init()
 
 #ifdef HAL_PERIPH_ENABLE_AIRSPEED
     if (airspeed.enabled()) {
+        airspeed.set_log_bit(false);
```

Comment:
false evaluates to zero and log bit 0 is a valid bit; I've used `-1` as a flag value in some places (e.g. `_log_raw_bit` in `AP_InertialSensor`)

This comes about because we were passing in a boolean from `should_log`....

Reply:
Sorry about that one.  I've left a comment instead so that if someone wants to enable airspeed logging it is little easier to figure out?